science/udunits2/Portfile

```diff
@@ -8,6 +8,11 @@ distname            udunits-${version}
 revision            0
 maintainers         {takeshi @tenomoto}
 license             Apache-2.0
+#                     The OSI-approved Apache-2.0 license is retroactively approved for
+#                     UDUNITS 2.2.28 by the upstream developer.  Please see:
+#                     https://github.com/Unidata/UDUNITS-2/issues/105#issuecomment-830422445
+#                     Please remove this comment on the next UDUNITS update that changes the
+#                     license embedded in the source code to Apache-2.0, as expected.
```

Comment:
I don't see the point of this, the license is already set to `Apache-2.0` so how does this comment help or do anything?